src/ape/api/compiler.py

```diff
@@ -166,6 +166,23 @@ def trace_source(  # type: ignore[empty-body]
             :class:`~ape.types.trace.SourceTraceback`
         """
 
+    def get_flattened_contract(self, path: Path, **kwargs) -> Content:
+        """
+        Get the content of a flattened contract via its source path.
+        Plugin implementations handle import resolution, SPX de-duplication,
```

Comment:
what does "SPX de-duplication" mean?

Reply:
for flattened contracts, you are only supposed to have 1 SPX identifier.
Sometimes, you may get 1 per source file and then you need to remove the duplicate ones.
is really just an example, just kinda explaining why the API generally does since it may seem like all you need to is paste the files on top of each other but actually is often more required steps (for Etherscan verification to be happy anyway)

Reply:
oh shit i meant SPDX

Reply:
ahhh